### PR TITLE
Update mailmap and re-generate Authors

### PR DESCRIPTION
**Pull request by thaJeztah**
Update mailmap file and re-generate authors; includes changes from https://github.com/docker/docker/pull/23168

closes https://github.com/docker/docker/pull/23168

### CODE REVIEW

**Comment by icecrime** [comment]
Oops, flaky test again? (cc @justincormack)

```
12:00:44 FAIL: docker_cli_run_unix_test.go:1089: DockerSuite.TestRunSeccompDefaultProfileNS
12:00:44 
12:00:44 docker_cli_run_unix_test.go:1148:
12:00:44     c.Assert(err, checker.IsNil)
12:00:44 ... value *errors.errorString = &errors.errorString{s:"goroutine 0: expected Operation not permitted, got: /go/src/github.com/docker/docker/bundles/1.12.0-dev/test-integration-cli/../binary-client/docker: Error response from daemon: Error processing tar file(exit status 1): open /etc/init.d/udev-finish: no space left on device.\nSee '/go/src/github.com/docker/docker/bundles/1.12.0-dev/test-integration-cli/../binary-client/docker run --help'.\n"} ("goroutine 0: expected Operation not permitted, got: /go/src/github.com/docker/docker/bundles/1.12.0-dev/test-integration-cli/../binary-client/docker: Error response from daemon: Error processing tar file(exit status 1): open /etc/init.d/udev-finish: no space left on device.\nSee '/go/src/github.com/docker/docker/bundles/1.12.0-dev/test-integration-cli/../binary-client/docker run --help'.\n")
```

File: .mailmap [hunk 1]
```diff
@@ -89,6 +89,7 @@ Jean-Baptiste Dalido <jeanbaptiste@appgratis.com>
 Sven Dowideit <SvenDowideit@home.org.au>
 Sven Dowideit <SvenDowideit@home.org.au> <SvenDowideit@fosiki.com>
 Sven Dowideit <SvenDowideit@home.org.au> <SvenDowideit@docker.com>
+Sven Dowideit <SvenDowideit@home.org.au> <¨SvenDowideit@home.org.au¨>
 Sven Dowideit <SvenDowideit@home.org.au> <SvenDowideit@home.org.au>
```
Comment (SvenDowideit) [inline .mailmap]: ew, sorry, idk where this comes from.

Comment (thaJeztah) [inline .mailmap]: @SvenDowideit Yes somewhere there's a commit with this, so adding it here prevents it from ending up in the Authors file :+1:

**Comment by vdemeester** [comment]
LGTM 🐮

**Comment by thaJeztah** [comment]
author LGTM (but I just spotted one duplicate, will do another one soon)

**Pull request merged**